CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 0.18.1
```

Comment:
Not necessarily for this PR, but we generally want to accumulate unpublished changes in a `-dev` version and then rev to the next appropriate stable version when we're ready to publish.

So here, we'd be accumulating the changes into `0.18.1-dev`, and rev to `0.18.1` when we're ready to publish. If one of those unpublished changes did happen to made some breaking API changes, we'd rev the major version of pending version (e.g., `0.19.0-dev`).